web/themes/custom/nysenate_theme/nysenate_theme.theme

```diff
@@ -4557,17 +4557,18 @@ function nysenate_theme_preprocess_node(&$variables) {
       $date = new \Datetime($node->field_date_range->value, new DateTimeZone('UTC'));
       $date->setTimezone(new DateTimeZone('America/New_York'));
       $date_end = new \Datetime($node->field_date_range->end_value, new DateTimeZone('UTC'));
```

Comment:
Why not set the correct timezone on instantiation?  Also examine getting the timezone, instead of hard-coding a default.

Reply:
this can be a separate ticket if you want fixed, unrelated to this ticket ... MC did this many times, at least 17 in theme file

Reply:
True, but we're here now  :)  
```$date - new \Datetime($node->field_date_range->value, new DateTimeZone('America/New_York'));```
instead of 
```$date = new \Datetime($node->field_date_range->value, new DateTimeZone('UTC'));
$date->setTimezone(new DateTimeZone('America/New_York'));```

Reply:
but none of this should be happening in the theme file to begin with!

Reply:
{{ content.field_date_range.value|date('U')|format_date('day_only') }}